libraries/HTTPUpdate/src/HTTPUpdate.h

```diff
@@ -89,6 +98,7 @@ class HTTPUpdate
     bool _rebootOnUpdate = true;
 private:
     int _httpClientTimeout;
+    followRedirects_t _followRedirects = HTTPC_DISABLE_FOLLOW_REDIRECTS;
```

Comment:
default value should be set in the class constructors instead.
```cpp
HTTPUpdate::HTTPUpdate(void)
        : _httpClientTimeout(8000), _followRedirects(HTTPC_DISABLE_FOLLOW_REDIRECTS), _ledPin(-1)
{
}
HTTPUpdate::HTTPUpdate(int httpClientTimeout)
        : _httpClientTimeout(httpClientTimeout), _followRedirects(HTTPC_DISABLE_FOLLOW_REDIRECTS), _ledPin(-1)
{
}
```

Reply:
Requested changes made.